### ISSUE TITLE
Got "SSL error: could not verify peer" when calling "travis branches" in the CI build script

### ISSUE THREAD

**Issue by weitjong** [travis-ci/travis.rb#730]
It was OK before. Checking the Issues history showed it has happened before in the past for other travis CLI command.

**Comment by BanzaiMan** [comment]
Is this still happening? If so, could you execute the same command with `--debug`?

**Comment by weitjong** [comment]
Yes, it is still happening. We call the CLI command inside our Travis CI job. The “SSL error“ can be seen in the log.

**Comment by BanzaiMan** [comment]
Which log?

**Comment by weitjong** [comment]
My apology for late response. Some how your mail got buried away.

You can find an example of the log at the end of the file here: https://travis-ci.org/github/urho3d/Urho3D/jobs/685250346.

**Comment by BanzaiMan** [comment]
Please add `--debug --debug-http` to the command to get more information on the failure. Thanks.
